integrationtests.py

```diff
@@ -40,12 +41,13 @@ def cd(targetDirectory):
 
 
 class TestCommandLineArguments(unittest.TestCase):
-    @unittest.skip("Do not run this test by default because it change user's cache settings")
     def testValidMaxSize(self):
-        validValues = ["1", "  10", "42  ", "22222222"]
-        for value in validValues:
-            cmd = CLCACHE_CMD + ["-M", value]
-            self.assertEqual(subprocess.call(cmd), 0, "Command must not fail for max size: '" + value + "'")
+        with tempfile.TemporaryDirectory() as tempDir:
+            validValues = ["1", "  10", "42  ", "22222222"]
+            for value in validValues:
+                cmd = CLCACHE_CMD + ["-M", value]
+                p = subprocess.Popen(cmd, env=dict(os.environ, CLCACHE_DIR=tempDir))
```

Comment:
Why plain Popen? All subprocess functions can be used with `env` as well

From https://docs.python.org/3/library/subprocess.html#subprocess.call

> The arguments shown above are merely the most common ones. The full function signature is largely the same as that of the Popen constructor - this function passes all supplied arguments other than timeout directly through to that interface.